### PR TITLE
[WIP] Renamed the repo and organized the repository structure

### PR DESCRIPTION
**Pull request by watakandai**
# Overview 
Renamed the entire repo and organized the repository structure

# Changes
- Renamed the repo
- Changed cod according to the new name
- Removed constant rotation data and the error function

# WIP
Need to run the script with data to test if it runs

### CODE REVIEW

File: robotic_skin/calibration/error_functions.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # import logging
 
 
-def max_angle_func(t: int, i_joint: int, delta_t=0.0, **kwargs):
+def max_angle_func(t: int, i_joint: int, delta_t=0.00, **kwargs):
```
Comment (peasant98) [inline robotic_skin/calibration/error_functions.py]: why the change?

File: robotic_skin/examples/calibration/plot_data.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 def fit_sin(t, y, rotate_joint):
     def optimize_func(x):
-        x[0]*np.sin(2*np.pi*C.PATTERN_FREQ[rotate_joint]*t+x[1]) + x[2] - y
+        return x[0]*np.sin(2*np.pi*C.PATTERN_FREQ[rotate_joint]*t+x[1]) + x[2] - y
```
Comment (peasant98) [inline robotic_skin/examples/calibration/plot_data.py]: this is just a stylistic comment, but the multiplication here looks too squashed together.

File: robotic_skin/calibration/error_functions.py (hunk 1 above)
Comment (watakandai) [reply robotic_skin/calibration/error_functions.py]: ah this is a mistake.

File: robotic_skin/examples/calibration/plot_data.py (hunk 2 above)
Comment (watakandai) [reply robotic_skin/examples/calibration/plot_data.py]: I guess I should use some variables to make this code easy to understand then.

**Review by peasant98 — APPROVED** [review]
